### PR TITLE
CLS statement

### PR DESCRIPTION
**Pull request by Esshahn**
Adds CLS statement as alias for ?CHR$(147)

### CODE REVIEW

File: Commander X16 Programmer's Reference Guide.md [hunk 1]
```diff
@@ -280,6 +281,17 @@ The string can contain printable ASCII characters (`CHR$($20)` to `CHR$($7E)`),
 	60 CHAR0,6+12*4,0,CHR$($0C)+A$ :REM OUTLINE
 	70 CHAR0,6+12*5,0,CHR$($12)+A$ :REM REVERSE
 
+#### CLS
+
+**TYPE: Command**
+**FORMAT: CLS**
+
+**Action:** Clears the screen. Same effect as `?CHR$(147)`.
```
Comment (greg-king5) [inline Commander X16 Programmer's Reference Guide.md]: Slight correction:  Same effect as `?CHR$(147);`.

(semicolon at the end)

Comment (Esshahn) [reply Commander X16 Programmer's Reference Guide.md]: interesting. why would the semicolon be relevant in this case? cheers!

Comment (greg-king5) [reply Commander X16 Programmer's Reference Guide.md]: `PRINT` automatically sends a carriage-return when it finishes, unless you tell it, "don't".  Putting a semicolon at the end of the statement tells `PRINT` to stay on the same line.

`CLS` leaves the cursor in the HOME position.  `?CHR$(147)` would put the cursor on the second line.

Comment (Esshahn) [reply Commander X16 Programmer's Reference Guide.md]: You're right! Great attention to detail, thanks.
Updated.

**Pull request merged**